pkg/base1/test-dbus-common.js

```diff
@@ -1050,7 +1050,7 @@ export function common_dbus_tests(channel_options, bus_name) { // eslint-disable
                 .always(function() {
                     assert.equal(this.state(), "resolved", "deleted stray objects");
 
-                    const proxies = dbus.proxies("com.redhat.Cockpit.DBusTests.Frobber");
+                    const proxies = dbus.proxies("com.redhat.Cockpit.DBusTests.Frobber", "/otree");
```

Comment:
Can we drop any conditional `.skip` with that change? ("test: Use ObjectManager in the D-Bus "proxies" test ")

Reply:
No, I think not. The other tests really want to test non-ObjectManager things.  We might never want to support that, but we shouldn't change the tests.